src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/utils/UserContextFilter.kt

```diff
@@ -0,0 +1,28 @@
+package uk.gov.justice.digital.hmpps.manageusersapi.utils
+
+import org.springframework.core.annotation.Order
+import org.springframework.http.HttpHeaders
+import org.springframework.stereotype.Component
+import java.io.IOException
+import javax.servlet.Filter
+import javax.servlet.FilterChain
+import javax.servlet.FilterConfig
+import javax.servlet.ServletException
+import javax.servlet.ServletRequest
+import javax.servlet.ServletResponse
+import javax.servlet.http.HttpServletRequest
+
+@Component
+@Order(4)
```

Comment:
I wonder what this annotation gives us?

Reply:
If I remember correctly Spring Security doesn't set an order on the Filter bean that it creates so by adding the order it just means they start the same each time, In some of the other projects there are more filters and I think one relied on an element in another so needed to be specifically ordered - will update this filter to number Order(2)